public/config/config.json

```diff
@@ -1,5 +1,5 @@
 {
   "apiBaseUrl": "http://127.0.0.1:8000/api/",
-  "dataSessionsUrl": "http://127.0.0.1:8000/api/datasessions/",
-  "dataLabArchiveUrl": "https://datalab-archive.photonranch.org/"
+  "dataLabArchiveApiUrl": "https://datalab-archive.photonranch.org/",
+  "ocsApiUrl": "https://observe.photonranch.org/api/"
```

Comment:
We might want to make this observationPortalApiUrl - OCS is a set of services, and we're specifically targeting the observation portal here